src/bitcoin-chainstate.cpp

```diff
@@ -0,0 +1,255 @@
+#include <iostream> // for cout and shit
+#include <functional> // for std::function
+#include <filesystem>
+
+#include <node/chainstate.h>          // for LoadChainstate
+#include <init/common.h>              // for SetGlobals, UnsetGlobals
+#include <validation.h>               // for ChainstateManager, InitScriptExecutionCache, StopScriptCheckWorkerThreads, UpdateUncommittedBlockStructures, BlockManager, DEFAULT_CHECKBLOCKS, DEFAULT_CHECKLEVEL
+#include <validationinterface.h>      // for GetMainSignals, cs_main, CMainSignals, RegisterSharedValidationInterface, UnregisterSharedValidationInterface, CValidationInterface
+#include <core_io.h>                  // for DecodeHexBlk
+#include <consensus/validation.h>     // for BlockValidationState
+#include <chainparams.h>              // for Params, SelectParams, CChainParams
+#include <node/blockstorage.h>        // for fReindex
+#include <util/system.h>              // for gArgs, ArgsManager
+#include <util/thread.h>              // for TraceThread
+#include <scheduler.h>                // for CScheduler
+#include <script/sigcache.h>          // for InitSignatureCache
+
+using node::fReindex;
+using node::LoadChainstate;
+using node::VerifyLoadedChainstate;
```

Comment:
In commit "build: Add example bitcoin-chainstate executable" (01840e6161396ed0ab30a122921f67b8dce45499)

I think it would be clearer to drop these using statements, and use the full names symbol below. Especially because `node::fIndex` looks like a local variable, not a global variable without the namespace component.

Reply:
Done as of 9050346f04